CONTRIBUTING.md

```diff
@@ -181,6 +181,19 @@ let's not fight over it.)
 
 We have an additional integration testing system in another repository: <https://github.com/FerretDB/dance>.
 
+#### Observability in tests
+
+Optionally, for those who prefer [OpenTelemetry tracing approach](https://opentelemetry.io/docs/concepts/signals/traces/),
+it is possible to use [Jaeger](https://www.jaegertracing.io/) in tests.
+
+Run `task jaeger` to start all-in-one Jaeger Docker container (the traces will be stored in memory and will be available
+while the container is running).
+
+Then run `task test-integration-pg JAEGER_ENDPOINT=http://localhost:14268/api/traces`
+to run `pg` handler tests with tracing enabled.
```

Comment:
Why is that configurable and not always enabled? It seems like that is always useful without even more knobs